pkg/services/libraryelements/database.go

```diff
@@ -120,12 +120,30 @@ func (l *LibraryElementService) createLibraryElement(c context.Context, signedIn
 			return model.LibraryElementDTO{}, model.ErrLibraryElementUIDTooLong
 		}
 	}
+
+	var modelMap map[string]interface{}
+	err := json.Unmarshal(cmd.Model, &modelMap)
+	if err != nil {
+		return model.LibraryElementDTO{}, err
+	}
+
+	if libraryPanel, ok := modelMap["libraryPanel"].(map[string]interface{}); ok {
+		if uid, ok := libraryPanel["uid"]; ok && uid == "" {
+			libraryPanel["uid"] = createUID
+		}
+	}
+
+	updatedModel, err := json.Marshal(modelMap)
+	if err != nil {
+		return model.LibraryElementDTO{}, err
+	}
+
```

Comment:
the library panel creation is storing a LibraryElement with a created UID, but the model inside the LibraryElement which is a LibraryPanel itself, doesn't have the id set (because it is bypassed based on the client data and there's no id). So, when retrieving this data, the LibraryPanel has a null id, which is wrong.